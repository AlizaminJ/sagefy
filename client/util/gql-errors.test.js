const getGqlErrors = require('./gql-errors')

describe('gql-errors', () => {
  it('should format errors', () => {
    expect(
      getGqlErrors({
        response: {
          errors: [{ message: 'email_check' }],
        },
      })
    ).toMatchSnapshot()
  })

  it('should format nothing', () => {
    expect(getGqlErrors({})).toEqual({})
  })
})
