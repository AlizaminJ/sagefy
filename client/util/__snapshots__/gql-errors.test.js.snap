// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gql-errors should format errors 1`] = `
Object {
  "email": Array [
    "Please format your email like \`a@b.c\`.",
  ],
}
`;
